CMakeLists.txt

```diff
@@ -636,7 +643,7 @@ if (ENABLE_KLEE_LIBCXX)
   # Find the library bitcode archive
   
   # Check for static first
-  set(KLEE_LIBCXX_BC_NAME "libc++.bca")
+  set(KLEE_LIBCXX_BC_NAME "libc++.bca") # for config.h
```

Comment:
Not used in code.

Reply:
Same here: https://github.com/klee/klee/blob/ba0a6136cf6bd446af64e66d8b21e8ba1018a96e/tools/klee/main.cpp#L1259